WhaleSpotting/Controllers/SightingsController.cs

```diff
@@ -33,11 +34,11 @@ public IActionResult CreateSighting([FromBody] SightingRequestModel sightingRequ
             {
                 var newSighting = _sightings.CreateSighting(sightingRequestModel);
                 return Created($"/sighting/{newSighting.Id}", newSighting);
-                // TODO note the url parameter above is to be updated if a get sighting by id endpoint is created. 
             }
             catch (Exception e)
             {
-                return BadRequest(e.Message);
+                ModelState.AddModelError("Thrown Error", e.Message);
```

Comment:
```suggestion
                ModelState.AddModelError(nameof(SightingRequestModel.SightedAt), e.Message);
```
We know what this error is on, so we should say that in the key

Reply:
Addressed.